dll/win32/shell32/dialogs/view.cpp

```diff
@@ -941,14 +951,26 @@ FolderOptionsViewDlg(
     switch (uMsg)
     {
         case WM_INITDIALOG:
-            return ViewDlg_OnInitDialog(hwndDlg);
+            return ViewDlg_OnInitDialog(hwndDlg, (PROPSHEETPAGE*)lParam);
 
         case WM_COMMAND:
             switch (LOWORD(wParam))
             {
                 case IDC_VIEW_RESTORE_DEFAULTS: // Restore Defaults
                     ViewDlg_RestoreDefaults(hwndDlg);
                     break;
+
+                case IDC_VIEW_APPLY_TO_ALL:
+                case IDC_VIEW_RESET_ALL:
+                    {
+                        HRESULT hr = HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED);
+                        CFolderOptions *pFO = (CFolderOptions*)GetWindowLongPtr(hwndDlg, GWL_USERDATA);
+                        if (pFO)
+                            hr = pFO->ApplyDefFolderSettings(LOWORD(wParam) == IDC_VIEW_RESET_ALL);
+                        if (FAILED(hr))
+                            SHELL_ErrorBox(hwndDlg, hr);
+                        break;
+                    }
```

Comment:
```suggestion
                {
                    HRESULT hr = HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED);
                    CFolderOptions *pFO = (CFolderOptions*)GetWindowLongPtr(hwndDlg, GWL_USERDATA);
                    if (pFO)
                        hr = pFO->ApplyDefFolderSettings(LOWORD(wParam) == IDC_VIEW_RESET_ALL);
                    if (FAILED(hr))
                        SHELL_ErrorBox(hwndDlg, hr);
                    break;
                }
```
https://reactos.org/wiki/Coding_Style#Indentation_and_line_width